### DISCUSSION TITLE
Question about the Etmp

### DISCUSSION THREAD

**Discussion by hansu1017** [PaddlePaddle/PaddleSpatial#92] [BaiduKDDCup2022] [answered]
Through data analysis, we found that the 'Etmp' can be higher than 100℃ in the full dataset, which  seemed to be abnormal. So can we deem these kind of samples as abnormal samples?

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

We do not intend to treat such value as abnormal value. It is possible for Etmp to be larger than 100℃. 

There may be a few values that are unreasonable, but not included in our rules to define abnormal values.  The reason is that there is no official rule to give a clear boundary to determine which values are normal and which are abnormal. But the percentage of such values is also very small.

However, you are welcome to give more cases for abnormal values, and we will carefully re-examine them.

Thanks.
